src/bucket.js

```diff
@@ -71,6 +81,7 @@ export default class Bucket {
    * @param  {Object} options.headers The headers object option.
    * @return {Promise<Object, Error>}
    */
+  @nobatch("Batch doesn't support read operations.")
```

Comment:
How would you consume the result of the read operation in the middle of a synchronous batch description as you suggested above? Please provide an example use case (eg. ask yourself "when do I get the actual results from my read operations?")